Traceback (most recent call last):
  File "c:\users\emsuewil\anaconda3\envs\stats_for_hds\lib\site-packages\jupyter_cache\executors\utils.py", line 56, in single_nb_execution
    record_timing=False,
  File "c:\users\emsuewil\anaconda3\envs\stats_for_hds\lib\site-packages\nbclient\client.py", line 1077, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
  File "c:\users\emsuewil\anaconda3\envs\stats_for_hds\lib\site-packages\nbclient\util.py", line 74, in wrapped
    return just_run(coro(*args, **kwargs))
  File "c:\users\emsuewil\anaconda3\envs\stats_for_hds\lib\site-packages\nbclient\util.py", line 53, in just_run
    return loop.run_until_complete(coro)
  File "c:\users\emsuewil\anaconda3\envs\stats_for_hds\lib\asyncio\base_events.py", line 573, in run_until_complete
    return future.result()
  File "c:\users\emsuewil\anaconda3\envs\stats_for_hds\lib\site-packages\nbclient\client.py", line 527, in async_execute
    cell, index, execution_count=self.code_cells_executed + 1
  File "c:\users\emsuewil\anaconda3\envs\stats_for_hds\lib\site-packages\nbclient\client.py", line 822, in async_execute_cell
    self._check_raise_for_error(cell, exec_reply)
  File "c:\users\emsuewil\anaconda3\envs\stats_for_hds\lib\site-packages\nbclient\client.py", line 730, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply['content'])
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
#Install package
#install.packages("VGAM")
#install.packages("pscl")
library("VGAM")
library(pscl)

#Number of data points to simulate
n=100
#Shape
lambda=3
#Probability of drawing a zero
p=0.5
#Create data
alcohol_count<-rzipois(n, lambda = lambda, pstr0 = p)
country <-  factor(sample(1:4, n, replace=T), levels=1:4, labels=c("England","Northern Ireland","Scotland","Wales"))
treatment <- factor(gl(2,n/2), levels=1:2, labels=c("Placebo Arm","Active Arm"))
print(df <- data.frame(treatment, country, alcohol_count))
summary(df)
summary(m1 <- zeroinfl(alcohol_count ~ country + treatment, data = df))
------------------

Error in library(pscl): there is no package called 'pscl'
Traceback:

1. library(pscl)
ERROR: Error in library(pscl): there is no package called 'pscl'


